webrender/src/render_backend.rs

```diff
@@ -426,12 +426,12 @@ impl RenderBackend {
                 }
                 None => {
                     // Was existing, not in current frame anymore
-                        let mut notifier = self.notifier.lock();
-                        notifier.as_mut()
-                                .unwrap()
-                                .as_mut()
-                                .unwrap()
-                                .pipeline_size_changed(pipeline_id, None);
+                    let mut notifier = self.notifier.lock();
+                    notifier.as_mut()
```

Comment:
I don't think this first `as_mut` is needed, and neither is the `let mut` thing. It is not related to the PR, but neither is this change related to freetype :)